Applications/WebPortal/WebApp/library/source/pages/group_page/group_info_page/group_info_page.tsx

```diff
@@ -15,38 +15,26 @@ interface Properties {
 export class GroupInfoPage extends React.Component<Properties> {
 
   public render(): JSX.Element {
-    const {displaySize, group} = this.props;
     const membersCount = (() => {
-      if(displaySize !== DisplaySize.SMALL) {
+      if(this.props.displaySize !== DisplaySize.SMALL) {
         return (
           <span style={GroupInfoPage.STYLE.membersInfo}>
-            Total members: {group.length}
+            Total members: {this.props.group.length}
           </span>);
       } else {
         return null;
       }
     })();
-    const pageWidth = (() => {
-      if(displaySize === DisplaySize.SMALL) {
-        return GroupInfoPage.STYLE.smallPageWidth;
-      } else if(displaySize === DisplaySize.MEDIUM) {
-        return GroupInfoPage.STYLE.mediumPageWidth;
-      } else {
-        return GroupInfoPage.STYLE.largePageWidth; 
-      }
-    })();
+    const pageWidth = GroupInfoPage.STYLE[this.props.displaySize];
```

Comment:
```pageWidth``` isn't used anywhere.

Reply:
It is.

`<div style={{...GroupInfoPage.STYLE.pageMargins, ...pageWidth}}>`